air/src/air/tests.rs

```diff
@@ -11,6 +11,10 @@ use crate::{FieldExtension, HashFunction};
 use crypto::{hashers::Blake3_256, RandomCoin};
 use math::{fields::f128::BaseElement, get_power_series, log2, polynom, FieldElement, StarkField};
 use rand::{seq::SliceRandom, thread_rng};
+
+#[cfg(feature = "alloc")]
+use crypto::hashbrown::HashMap;
+#[cfg(feature = "std")]
```

Comment:
Should this also be replaced with `use utils::collections::HashMap`

Reply:
That's actually a miss! Thank you for catching it! (tests have not been made no-std, so technically we could just use `use std::collections::HashMap`). I can try moving tests as well to `no-std` if wanted.

Reply:
Note that this would imply to make the `rand()` method of the `FieldElement` trait accessible in `no-std` as well (currently hidden behind the `std` feature